src/Upgrade/Application/Strategy/Composer/Fixer/FeatureDevMasterPackageFixerStep.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace Upgrade\Application\Strategy\Composer\Fixer;
+
+use Upgrade\Application\Adapter\PackageManagerAdapterInterface;
+use Upgrade\Application\Dto\StepsResponseDto;
+use Upgrade\Domain\Entity\Collection\PackageCollection;
+use Upgrade\Domain\Entity\Package;
+
+class FeatureDevMasterPackageFixerStep extends AbstractFeaturePackageFixerStep
+{
+    /**
+     * @var string
+     */
+    public const ALIAS_DEV_MASTER = 'dev-master as 202000.0';
```

Comment:
As far i remember the first 4 numbers should be generated from the current date. Pls clarify with PO